### DISCUSSION TITLE
Active connections not displayed when using port 22 with iptables

### DISCUSSION THREAD

**Discussion by mezuky04** [drakkan/sftpgo#1095] [General]
Hello. I'm using the following rules to forward port 2022 to 22 (taken from the docs) and it works as expected. 
```shell
sudo iptables -t nat -A PREROUTING -d <ip> -p tcp --dport 22 -m addrtype --dst-type LOCAL -j DNAT --to-destination <ip>:2022
sudo iptables -t nat -A OUTPUT     -d <ip> -p tcp --dport 22 -m addrtype --dst-type LOCAL -j DNAT --to-destination <ip>:2022
```

However, if I connect via SFTP using port 22 the connection is not displayed in the connections page. If I connect using port 2022, the active connection is displayed.

Any idea how to display the active connections on port 22? Thanks 🙂

**Comment by drakkan** [comment]
Hello, please check your logs, if you see a login in the SFTPGo logs you should also see an active connection

**Comment by mezuky04** [comment]
I checked the logs and there is an entry each time the user logs in, no matter if port 2022 or 22 is used. However, the active connection is displayed only when port 2022 is used. I have two SFTPGo instances using a single CockroachDB as a data store.

**Reply by drakkan** [reply]
> I checked the logs and there is an entry each time the user logs in, no matter if port 2022 or 22 is used. However, the active connection is displayed only when port 2022 is used. I have two SFTPGo instances using a single CockroachDB as a data store.

This is strange, are you listing active connections from the same SFTPGo instance that the client connects to?

**Comment by mezuky04** [comment]
That's the problem. I wrongly thought that I can make the API requests to any instance and will return all the connections since the same data store is used. Thanks for your time 🙂

**Reply by drakkan** [reply]
> That's the problem. I wrongly thought that I can make the API requests to any instance and will return all the connections since the same data store is used. Thanks for your time slightly_smiling_face

It is supported but you must configure SFTPGo appropriately otherwise instance1 doesn't know that instance2 exists. Check the [docs](https://github.com/drakkan/sftpgo/blob/main/docs/full-configuration.md) expand "Configuration file" and search for "node". You also have to set `is_shared` to `1`

**Comment by mezuky04** [comment]
It works as expected with the `node` & `is_shared` options. Thanks 🙂
